.github/workflows/build.yml

```diff
@@ -148,6 +148,11 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ inputs.configure-aws-region }}
+      - name: Login to Docker
```

Comment:
I would make it an optional step with
if: ${{ inputs.login-to-docker }}
default can be true, but I can imagine some builds where you don't want to do this as we won't use docket. So it would be nice if you can turn this off.

examples:
inputs.create-npmrc or inputs.create-jfrog-npmrc

Reply:
Thanks. That makes sense, Have made that change now.